src/types/web3/eth/index.d.ts

```diff
@@ -22,7 +22,7 @@ import {
 type GetPastLogsOpts = {
   fromBlock?: BlockType;
   toBlock?: BlockType;
-  address: string;
+  address?: string;
```

Comment:
Why was this made optional?

Reply:
It was a bug in the pre-existing web3 typings. The address is optional!